packages/react-components/src/trace-explorer/trace-explorer-views-widget.tsx

```diff
@@ -171,7 +171,7 @@ export class ReactAvailableViewsWidget extends React.Component<ReactAvailableVie
         if (descriptors && descriptors.length) {
             outputDescriptors.push(...descriptors);
         }
-        return outputDescriptors.filter(value => value.type !== 'DATA_TREE');
+        return outputDescriptors.filter(value => value.type !== 'DATA_TREE' && value.type !== 'TREE_TIME_XY');
     }
```

Comment:
I'm against removing the XY charts as a whole. Event if they have short-comings they have already value, e.g. CPU usage view or histogram. Besides the charts are being improved right now regarding navigation. I suggest to fix the most pressing short-comings (e.g. y-labels, tooltips) instead of hiding them.

Reply:
What is a concrete example of the value they offer without having any sense of scale? I gave this some thought before proposing the patch and didn't have any luck:
* Since the data is displayed in a line you can see the shape of the data, which in theory could allow you to use this as a reference for where to explore more in detail using a timegraph chart, but without scale there's no way of knowing whether spikes in the graph are just noise. 
* I could imagine an experienced trace compass user might know whether the shape of a graph looks typical or not in specific situations, but this feels like a stretch.

Reply:
@ebugden I suggest that the pressing short-comings are fixed instead of removing the whole support of such graphs.